package.json

```diff
@@ -102,6 +102,7 @@
         "electron-devtools-installer": "3.2.0",
         "electron-notarize": "0.3.0",
         "esbuild": "0.14.49",
+        "esbuild-sass-plugin": "^2.4.0",
```

Comment:
Shall we pin the version as the other dependencies?

Reply:
Removed it alltogether, getting it from shared.